src/main/java/com/kk/heapoverflow/controller/QuestionController.java

```diff
@@ -0,0 +1,45 @@
+package com.kk.heapoverflow.controller;
+
+
+import com.kk.heapoverflow.service.*;
+import lombok.*;
+import lombok.extern.slf4j.*;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/questions")
```

Comment:
see `application.properties`
```suggestion
@RequestMapping("/questions")
```